ConfigurationSystem/Client/Utilities.py

```diff
@@ -273,6 +273,13 @@ def addToChangeSet( entry, changeSet ):
           newMaxCPUTime = queueInfo.get( 'GlueCEPolicyMaxCPUTime', '' )
           if newMaxCPUTime == "4" * len( newMaxCPUTime ) or newMaxCPUTime == "9" * len( newMaxCPUTime ):
             newMaxCPUTime = ''
```

Comment:
I know the 2 lines above are not part of the changes of this PR, but... what the heck is that?